src/System.Private.CoreLib/src/System/Runtime/Bootstrap/BootstrapHelpers.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using Internal.NativeFormat;
+using System.Runtime.Bootstrap;
+
+namespace System.Runtime
+{
+    internal static class BootstrapHelpers
+    {
+        internal static IntPtr s_strEETypePtr = IntPtr.Zero;
+        internal static IntPtr s_strTableStart = IntPtr.Zero;
+        internal static IntPtr s_strTableEnd = IntPtr.Zero;
+
+        internal enum ModuleSectionIds
+        {
+            StringEETypePtr,
+            StringFixupStart
+        };
+
+        internal static void Initialize()
+        {
+            InitializeStringTable();
+        }
+
+        internal static void Shutdown()
+        {
+
+        }
+
+        internal static string[] GetCommandLineArgs(int argc, IntPtr argv)
+        {
+            int length = 0;
+            System.Diagnostics.Debug.Assert(length == IntPtr.Size);
+            return StringFixup.GetCommandLine(argc, argv, s_strEETypePtr);
```

Comment:
The code to create the string array should be here, not on the StringFixup. And it should use regular new String(...) because of the runtime is initialized by the time you get here.